### PR TITLE
Extracting store and services

### PR DESCRIPTION
**Pull request by chrisdugne**
**Detailed purpose of the PR**

app-player store is currently not reusable as standalone

```
app-player
├── store
│   ├── actions
│   ├── middlewares
│   ├── reducers
│   ├── services
│   └── view
│       └── state-to-props
└── web
    └── views
```

**Result and observation**

app-player was split in order to use `store` and `services` in other projects
```
player-services
player-store
├── actions
├── middlewares
└── reducers
player-web
├── map-state-to-props
└── views
```

notes to help reviewers: 
https://github.com/CoorpAcademy/components/pull/1399#pullrequestreview-179887349
https://github.com/CoorpAcademy/components/pull/1399#pullrequestreview-179888184

- [x]  **Breaking changes ?**  
If checked, what have you broken ?

`app-player` is removed.
`player-web` is to be used from now on.

- [ ] **Extra lib ?** 
If checked, Which extra lib did you add ? (name, purpose, link ...).

**How tested ?**
 - [x] Already covered by tests
 - [x] Manual testing
 - [x] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-player-web/src/index.js [hunk 1]
```diff
@@ -30,14 +30,16 @@ const create = options => {
 
   /* istanbul ignore if  */
   if (module.hot) {
-    module.hot.accept('../store/view', () => {
```
Comment (godu) [inline packages/@coorpacademy-player-web/src/index.js]: ça peut poser problème ce genre de require...

Comment (chrisdugne) [reply packages/@coorpacademy-player-web/src/index.js]: fixed

**Comment by chrisdugne** [comment]
done @godu

**Comment by chrisdugne** [comment]
[extracting player-store and player-service](https://trello.com/c/rFTfJ0jp/1017-extracting-player-store-and-player-service)

**Comment by chrisdugne** [comment]
done @godu

**Pull request merged**